src/dbSta/test/hierclock_out.vok

```diff
@@ -83,29 +83,29 @@ module hierclock (a_count_valid_o,
     .rst_n_i(rst_n_i),
     .clk_i(clk_i));
  counter U2 (.count_valid_o(a_count_valid_o),
+    .\count_value_o[3] (a_count_o[3]),
+    .\count_value_o[2] (a_count_o[2]),
+    .\count_value_o[1] (a_count_o[1]),
+    .\count_value_o[0] (a_count_o[0]),
+    .\load_value_i[3] (a_i[3]),
+    .\load_value_i[2] (a_i[2]),
+    .\load_value_i[1] (a_i[1]),
+    .\load_value_i[0] (a_i[0]),
```

Comment:
why did this change? seems like you would want to keep ports together like before.

Reply:
Thanks for the comment. You are absolutely right !  Support for busses would be added in the next pull request. This draft pull request is to validate the idea of using odb natively for hierarchy without an intermediate "fake library". Just to recap the "fake library" is created in dbNetwork.cc in the current (master) code for both hierarchy cells and liberty cells. But it was felt that maintaining the fake library for hierarchy was wrong (eg adding a port to some intermediate hierarchical level would also requiring updating the fake library). You are absolutely right that the current master code actually supports busses on the fake library and I believe it will also be possible to support them on the "native odb approach". However, before doing the work on the busses I want to put the code out there for the native odb approach to review with Matt. One of the things I am a bit worried about is that I have to keep notes of the "fake library cells" for liberty so that the type conversion (reinterpret_cast) stuff is never given something it cannot convert. If that is acceptable then I will proceed with folding in the bus support. But first I wanted to see if this concept of having fake library for Liberty (needed for timer) but native support for hierarchy on odb is cool.